10/umbraco-cms/implementation/controllers.md

```diff
@@ -52,8 +52,34 @@ An Umbraco Authorized Controller is used when the controller requires member or
 The Umbraco Authorized controllers and attributes for Backoffice Users are:
 
 *   **MVC**
+    There is no specific controller available to inherit from. We recommend inheriting from the basic `Umbraco.Cms.Web.Common.Controllers.UmbracController`` and applying the following attributes to your method
+    - `[Authorize(Policy = AuthorizationPolicies.BackOfficeAccess)]`: Uses .net authorization using the BackOfficeAccess policy. We recommend adding extra custom authorization policies for your endpoints.
+    - `[DisableBrowserCache]`: Tells the browser to not cache the result.
 
-    A base class implementation for backoffice authorized controllers is inherited from: `Umbraco.Cms.Web.Common.Controllers.UmbracoAuthorizedController`. These MVC controllers are not auto-routed.
+    Remember to [add a route](../reference/routing/authorized/defining-a-route) for your controller
+
+    ###### Example custom authorized backoffice mvc controller
```

Comment:
previous sections has 3 ### so as this is an example part of same section it should have only 4 ####
```suggestion
    #### Example custom authorized backoffice MVC Controller
```